### ISSUE TITLE
Screen readers should read the names of labels when tabbing in a label track

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#1838]
Now that when a label track is the focus tab and shift+tab only "select" a label, rather than opening it for editing in a custom text box which is not accessible for screen readers, it's useful for the name of the label to be read.

Pull request arriving shortly.
